_Week5/group4/README.md

```diff
@@ -0,0 +1,16 @@
+# Week 5 Homework Assignment, Group 4: `graphsum`
+
+## Overview
+
+`graphsum` creates basic graphs and summary statistics for two variables in a system dataset. It takes in 3 inputs—`sysdata`, `var1`, and `var2`—and outputs 4 graphs and 3 matrices. It loads the system data called ``sysdata’.dta` and it creates a scatter plot with a fitted line where `var1` is the dependent variable and `var2` is the independent variable, a histogram of `var1`, a histogram of `var2`, and a combined graph with both histograms side by side. The graphs are stored in the working directory. It also produces summary statistics for `var1` and `var2` and stores them in matrices, and it runs a robust regression of `var1` on `var2` and stores the coefficient matrix.
```

Comment:
Nice! What variable formats does the program accept? Numeric? Strings? Encoded categorical?